src/Tribe/Views/V2/Widgets/Widget_QR_Code.php

```diff
@@ -235,10 +233,18 @@ public function setup_admin_fields() {
 			];
 
 			$series = get_posts( $args );
-			foreach ( $series as $series ) {
+			if ( ! empty( $series ) ) {
+				$series_options = [];
+				foreach ( $series as $series ) {
```

Comment:
This isn't new to your code, but now's the time to fix this: The variable is the same on both sides of `as`. The second variable should be changed.